docs/semgrep-ci/configuration-reference.md

```diff
@@ -29,17 +29,13 @@ NOTE: It is recommended to perform baseline scans on other branches than your `m
 
 To only report findings newly added
 since branching off from your `main` branch, set the following:
-```sh
-SEMGREP_BASELINE_REF=$(git merge-base main HEAD)
-```
-
-NOTE: Because of an issue, the option `SEMGREP_BASELINE_REF=main` does not work correctly. The `SEMGREP_BASELINE_REF=$(git merge-base main HEAD)` is a workaround that allows Semgrep to only report the findings made on a topic branch after it diverged from the `main` branch. 
+<pre class="language-bash"><code>SEMGREP_BASELINE_REF=<span className="placeholder">TOPIC-BRANCH-NAME</span></code></pre>
 
 To only report findings newly added
 after a specific commit, set the following:
 <pre class="language-bash"><code>SEMGREP_BASELINE_REF=<span className="placeholder">INSERT_GIT_COMMIT_HASH</span></code></pre>
 
-Instead of `SEMGREP_RULES`, configure which rules to run with Semgrep App.
+TODO Instead of `SEMGREP_RULES`, configure which rules to run with Semgrep App.
```

Comment:
Why are we introducing this just after baselining? This is a separate thing, right? Is this a promotion for Semgrep App or any specific part of a different section that somehow ended up here? :-)

Reply:
Yeah this looks really weird and out of place.